diff -up xmlrpc-epi-0.54.2/src/xmlrpc_introspection.c.omv~ xmlrpc-epi-0.54.2/src/xmlrpc_introspection.c
--- xmlrpc-epi-0.54.2/src/xmlrpc_introspection.c.omv~	2018-04-30 11:53:57.324987750 +0200
+++ xmlrpc-epi-0.54.2/src/xmlrpc_introspection.c	2018-04-30 11:54:11.508657306 +0200
@@ -78,7 +78,7 @@ int validate_value( XMLRPC_VALUE xDesc,
 /* iterates through a list of structs and finds the one with key "name" matching
  * needle.  slow, would benefit from a struct key hash.
  */
-inline XMLRPC_VALUE find_named_value(XMLRPC_VALUE list, const char* needle) {
+static inline XMLRPC_VALUE find_named_value(XMLRPC_VALUE list, const char* needle) {
    XMLRPC_VALUE xIter = XMLRPC_VectorRewind(list);
    while(xIter) {
       const char* name = XMLRPC_VectorGetStringWithID(xIter, xi_token_name);
@@ -108,7 +108,7 @@ static void check_docs_loaded(XMLRPC_SER
 
 
 /* utility function for xi_system_describe_methods_cb */
-inline void describe_method(XMLRPC_SERVER server, XMLRPC_VALUE vector, const char* method) {
+static inline void describe_method(XMLRPC_SERVER server, XMLRPC_VALUE vector, const char* method) {
    if(method) {
       server_method* sm = find_method(server, method);
       if(sm) {
